webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -39,6 +44,31 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
     setIsShown(!isShown)
   }
 
+  const updateMultiImgHeight = (img: HTMLImageElement) => {
+    const aspectRatio = img.naturalWidth / img.naturalHeight
+    const width = img.clientWidth
+    const calculatedHeight = Number.parseFloat((width / aspectRatio).toFixed(2))
+    setMultiImgHeight(calculatedHeight)
+  }
+
+  useEffect(() => {
```

Comment:
Tried adding a test but it would fail:

```
  it('should update the multi image height when changing the width of plots', async () => {
    const store = renderAppWithOptionalData({
      comparison: comparisonTableFixture
    })
    setWrapperSize(store)

    const multiImage = screen.getAllByTestId('multi-image-cell')[0]

    const originalHeight =
      getComputedStyle(multiImage).getPropertyValue('--img-height')

    await waitFor(
      () => {
        const imgEl: HTMLImageElement = within(multiImage).getByRole('img')
        expect(Boolean(imgEl.complete)).toStrictEqual(true) // fails
      },
      { timeout: 1000 }
    )

    sendSetDataMessage({
      comparison: {
        ...comparisonTableFixture,
        width: 3
      }
    })

    await waitFor(
      () => {
        expect(
          getComputedStyle(multiImage).getPropertyValue('--img-height')
        ).not.toBe(originalHeight)
      },
      { timeout: 1000 }
    )
  })
```

I think it's due to how the virtual dom deals with images 🤔. Any idea on how to test for this?

Reply:
Have to take a snapshot

Reply:
Added an extra snapshot!